### PR TITLE
Fix error with webpack 2

### LINKED ISSUE #29: updates for webpack 2?
Just wondering if there is a plan to update this plugin for webpack 2.  Thanks.

### PR DESCRIPTION
**Pull request by isaacplmann**
This should still work with webpack 1 as well.

Fixes #29... I think.

### CODE REVIEW

**Comment by isaacplmann** [comment]
With this fix I can build my application with webpack 2.

I'm also fairly confident it will work in webpack 1 since the modified line of code is the same as is used in extract-text-webpack-plugin that works for both webpack 1 and 2. 
Here's the relevant line: https://github.com/webpack/extract-text-webpack-plugin/blob/1a29dd60554f63c39d23b0ea11ee242fe7a107da/index.js#L6

**Comment by isaacplmann** [comment]
And here's the webpack 1 version of extract-text-webpack-plugin:
https://github.com/webpack/extract-text-webpack-plugin/blob/webpack-1/index.js#L5

File: ProductionModePlugin.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 var CommonJsRequireDependency = require("webpack/lib/dependencies/CommonJsRequireDependency");
-var ConcatSource = require("webpack/lib/ConcatSource");
+var ConcatSource = require("webpack-sources").ConcatSource;
```
Comment (rxaviers) [inline ProductionModePlugin.js]: If using `require("webpack-sources")`, this needs to be declared as a `dependency` in `package.json`.

Comment (rxaviers) [inline ProductionModePlugin.js]: How does `webpack@1.9.0/lib/ConcatSource` compare to `webpack-sources` please? Is it equivalent or are there any changes? Thanks

**Comment by isaacplmann** [comment]
webpack-sources ConcatSource has a pretty small [API](https://github.com/webpack/webpack-sources#concatsource).  It looks like it has all the same public functions as [the webpack 1 version](https://github.com/webpack/core/blob/master/lib/ConcatSource.js).

**Comment by rxaviers** [comment]
Thanks for the updates.

**Comment by rxaviers** [comment]
For the record, if one installs globalize-webpack-plugin today with the latest webpack@1.x, the result is `webpack@1.13.2` along with `webpack-core@0.6.8`. Also, `require("webpack/lib/ConcatSource")` = `require("webpack-core/lib/ConcatSource")`. Comparing `ConcatSource` between `webpack-core@0.6.8` and `webpack-sources@0.1.2` we get virtually the same file: https://gist.github.com/rxaviers/6b9e0216932100a1e83241c26aeffdf3. Therefore, this change has zero side-effect for users at the latest webpack 1.x. Also the only change that happened on `ConcatSource` since this plugin exists is https://github.com/webpack/core/commit/90790fc74d4f6aa4a5985673b4e0f7e128645052#diff-083b45da4356d4db2836b27f118f22bc, so I believe it's safe to release this change as a patch release.

**Comment by rxaviers** [comment]
Hey @isaacplmann, I need one more thing from you, sorry for not having pointed it out earlier. I need you to update the commit using your real name and to sign-off DCO. Basically:
- https://gist.github.com/yano3/1378948, and
- http://stackoverflow.com/a/15667644/798133

Sorry for the delay in the ping pong.

**Comment by isaacplmann** [comment]
I think that did it.

**Comment by rxaviers** [comment]
Yay! Thanks. Released as `v0.3.6`.

**Pull request closed without merging**